src/mime.cr

```diff
@@ -0,0 +1,167 @@
+require "crystal/system/mime"
+
+# This module implements a global MIME registry.
+#
+# ```
+# require "mime"
+# MIME[".html"] # => "text/html"
```

Comment:
Is `MIME.[]` implemented?

Reply:
Ah, no. I was unsure about that. Should we have `MIME.[]` and/or `MIME.fetch`? Or a different name?

Reply:
Personally, I'm finding `Klass.[]` aliased to `Klass.fetch` as a quite nice syntactic sugar, but of course there's the Crystal stand on aliasing...

Reply:
No aliases please. Let's just name these methods normally and leave aside the unnecessary syntax sugar.

Reply:
Yeah, I figure it doesn't make much sense here anyway.